UserInterface/MainWindow.cpp

```diff
@@ -534,6 +535,19 @@ void OpenInfraPlatform::UserInterface::MainWindow::updateModelsUI()
 					// do nothing
 				}
 
+				// 5. treeviewer
+				QString filetype = filename.right(3);

```

Comment:
I think that the variable `filetype` is not needed anymore.

Reply:
Resolved: 6de63e9